test/unit/lib/plugins/package/lib/packageService.test.js

```diff
@@ -399,6 +31,8 @@ describe('lib/plugins/package/lib/packageService.test.js', () => {
 
   describe('service wide', () => {
```

Comment:
I think in the context of this specific describe, it would be better to, for most of the tests, assert that `artifact.zip` that is packaged includes (or not includes) specific files, instead of always checking on `fnIndividual`'s artifact that was packaged separately. Still, the tests that check `individually: true` flag should stick to using it, but for others, I think we should unzip the `service-wide` artifact and inspect it's contents. Sorry for not catching it sooner, I think I've missed that suggestion before

Reply:
I'm not sure if I understand. Currently in `service wide` artifact is not specified here so includes and excludes are ignored. Did you mean `service-wide` introduced in 0f7b31b?

Reply:
Sorry, I wasn't 100% clear in my previous message. I believe this particular `describe` section should concern `artifact` that is produced for the whole service (so not artifact that is only for specific individually packaged function). For example, if we have a few functions, they should be all packaged under one zip, which is named after `name` of the `service` from `serverless.yml`. In case of this particular fixture, `package.artifact` is already defined, so no other `service-wide` artifact will be created, but I think the fixture here should be configured in a way that `package` run will produce this `service-wide` artifact and we should perform assertions against files that ended up in this `service-wide` artifact zip file.

However, I'm not longer 100% sure about the approach that was intended here, @medikoo, could you confirm if my thinking above is correct or it's fine to perform all assertions in tests against files that were packaged for the `fnIndividual` function?